interpreter/src/main/scala/sigmastate/crypto/Platform.scala

```diff
@@ -94,6 +105,9 @@ object Platform {
   /** Negate a point. */
   def negatePoint(p: Ecp): Ecp = Ecp(p.value.negate())
 
+  /** Wrapper for curve type. */
```

Comment:
But why the wrapper is needed at all ?

Reply:
To hide underlying nature of the abstraction and avoid leakage of the concrete ECCurve class